de.marlam.qv.yml

```diff
@@ -0,0 +1,166 @@
+app-id: de.marlam.qv
+runtime: org.kde.Platform
+runtime-version: '6.4'
+sdk: org.kde.Sdk
+command: qv
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # OpenGL rendering
+  - --device=dri
+  # Read any scientific data file (mostly stored outside the xdg hierarchy)
+  # and save a visualization alongside it
+  - --filesystem=host
+cleanup:
+  # '/bin/[a-p]*' does not work, so list explicitly
+  - '/bin/M*'
+  - '/bin/a*'
+  - '/bin/c*'
+  - '/bin/d*'
+  - '/bin/e*'
+  - '/bin/g*'
+  - '/bin/h*'
+  - '/bin/i*'
+  - '/bin/j*'
+  - '/bin/m*'
+  - '/bin/n*'
+  - '/bin/o*'
+  - '/bin/p*'
+  - '/bin/r*'
+  - '/bin/s*'
+  - '/bin/t*'
+  - '/bin/x*'
+  - '/include'
+  - '/lib/cfitsio*'
+  - '/lib/cmake'
+  - '/lib/pkgconfig'
+  - '/lib/python*'
+  - '/lib/*.a'
+  - '/share/COPYING'
+  - '/share/RELEASE.txt'
+  - '/share/USING_HDF5_CMake.txt'
+  - '/share/bash-completion'
+  - '/share/cmake'
+  - '/share/doc'
+  - '/share/man'
+modules:
+  # Exiv2: used by libtgd to read png and jpeg metadata
+  - name: exiv2
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/Exiv2/exiv2/releases/download/v0.27.5/exiv2-0.27.5-Source.tar.gz
+        sha256: 35a58618ab236a901ca4928b0ad8b31007ebdc0386d904409d825024e45ea6e2
+  # Dcmtk: used by libtgd to read DICOM medical image data
+  - name: dcmtk
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DBUILD_APPS=OFF
+      - -DBUILD_SHARED_LIBS=ON
+    sources:
+      - type: archive
+        url: https://dicom.offis.de/download/dcmtk/dcmtk367/dcmtk-3.6.7.tar.gz
+        sha256: 7c58298e3e8d60232ee6fc8408cfadd14463cc11a3c4ca4c59af5988c7e9710a
+  # Imath: prerequisite for OpenEXR
+  - name: Imath
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/AcademySoftwareFoundation/Imath/archive/refs/tags/v3.1.6.tar.gz
+        sha256: ea5592230f5ab917bea3ceab266cf38eb4aa4a523078d46eac0f5a89c52304db
+  # OpenEXR: used by libtgd to read EXR image data
+  - name: OpenEXR
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/AcademySoftwareFoundation/openexr/archive/refs/tags/v3.1.5.tar.gz
+        sha256: 93925805c1fc4f8162b35f0ae109c4a75344e6decae5a240afdfce25f8a433ec
+  # cfitsio: used by libtgd to read FITS astronomy data
+  - name: cfitsio
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: http://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-4.2.0.tar.gz
+        sha256: eba53d1b3f6e345632bb09a7b752ec7ced3d63ec5153a848380f3880c5d61889
+  # poppler: used by libtgd to read PDFs
+  - name: poppler
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DENABLE_BOOST=OFF
```

Comment:
so I assume this doesn't use the splash backend.

Reply:
It does; boost is said to be recommended for better performance, but it actually builds without boost, and I left it out to not add to the already long list of dependencies.

Reply:
then you should add boost. You only need the headers.